src/wffile.c

```diff
@@ -105,6 +105,12 @@ OpenFileForCompress(
     PHANDLE phFile,
     LPTSTR szFile);
 
+DWORD
+WFCopySymlink(
+   LPTSTR pszFrom,
+   LPTSTR pszTo,
+   DWORD dwFlags,
+   DWORD dwNotification);
 

```

Comment:
This should go in winfile.h; the other declarations above this are for functions in this file.